hw-05/Models/Animal.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Models
+{
+    public class Animal
+    {
+        public Animal(string name, int age, double size)
+        {
+            _name = name;
+            _age = age;
+            _size = size;
+        }
+        public static string _name;
+        public static int _age;
+        public static double _size;
```

Comment:
```suggestion
        public string _name;
        public int _age;
        public double _size;
```
имя, возраст, размер - это данные про экземпляр, про конктрный инстанс
поэтому они не должны быть статичными, а должны быть обычными полями